src/bundle/Resources/views/content/tabs/versions/table.html.twig

```diff
@@ -0,0 +1,32 @@
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{'locationview.versions.version'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.modified.language'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.author'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.created'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.last.saved'|trans({}, 'locationview')}}</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for version in versions %}
+        <tr>
+            <td>
+                {{ version.versionNo}}
+            </td>
+            <td>
+                {{ version.getContentInfo().mainLanguageCode }}
+            </td>
+            <td>
+                {{ ez_render_field(authors[version.id], 'first_name') }} {{ ez_render_field(authors[version.id], 'last_name') }}
```

Comment:
same remark as in https://github.com/ezsystems/hybrid-platform-ui/pull/18#discussion_r122197912, `first_name` and `last_name` fields might not be there, `{{ authors[version.id].name }}` should display the user full name